### DISCUSSION TITLE
Cannot run SVCG

### DISCUSSION THREAD

**Discussion by AlbertoCapraro** [voicepaw/so-vits-svc-fork#827] [Q&A]
Hi! 

I got the project cloned by using Pycharm. The installation procedure runs correctly, but when I try to run the GUI interface I get the following message: ( after running svcg on Python console, but I've tried on terminal as well )

PyDev console: starting.
Python 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)] on win32
svcg
Traceback (most recent call last):
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2023.1.3\plugins\python-ce\helpers\pydev\pydevconsole.py", line 364, in runcode
    coro = func()
  File "<input>", line 1, in <module>
NameError: name 'svcg' is not defined

What am I doing wrong? I tried many options, changing enviroments also, but I've run out of options.

Thanks to everyone

Regards,
Alberto

**Comment by RudivanHemert** [comment]
The error message "name 'svcg' is not defined" means that the Python interpreter cannot find the svcg module. This could be because the module is not installed, or because it is not in the Python path.

To check if the svcg module is installed, you can run the following command in a terminal:

```
pip show svcg
```

If the module is installed, you should see some output about the module, including its version number. If the module is not installed, you will see an error message.

To install the svcg module, you can run the following command in a terminal:

```
pip install svcg
```

Once the svcg module is installed, you should be able to run it by typing the following command in a terminal:

```
svcg
```

If you are still getting the error message, it is possible that the svcg module is not in the Python path. The Python path is a list of directories that Python searches for modules. To add the svcg module to the Python path, you can run the following command in a terminal:


`export PYTHONPATH=$PYTHONPATH:/path/to/svcg
`

Replace `/path/to/svcg` with the path to the directory where the svcg module is installed.

Once you have added the svcg module to the Python path, you should be able to run it without any problems.

Here are some additional things you can try:

* Make sure that you are using the correct version of Python. The svcg module requires Python 3.6 or higher.
* Try running the svcg module from a different directory.
* Try restarting your computer.

If you are still having problems, you can ask for help on the so-vits-vsc-fork GitHub page.
